backend/packages/Upgrade/src/api/services/FeatureFlagService.ts

```diff
@@ -64,6 +65,21 @@ export class FeatureFlagService {
 
     const includedFeatureFlags = await this.featureFlagLevelInclusionExclusion(filteredFeatureFlags, experimentUserDoc);
 
+    // save exposures in db
+    try {
+      const exposureRepo = getRepository(FeatureFlagExposure);
+
+      await Promise.all(
+        includedFeatureFlags.map((flag) => {
+          return exposureRepo.save({ featureFlag: flag, experimentUser: experimentUserDoc });
+        })
+      );
+    } catch (err) {
+      const error = new Error(`Error in saving feature flag exposure records ${err}`);
+      (error as any).type = SERVER_ERROR.QUERY_FAILED;
+      throw error;
```

Comment:
Should we throw an error here or log the error and complete the request?
Saving the exposure is not a primary action, this HTTP request should succeed.